tasks/cea-install.js

```diff
@@ -1,30 +1,82 @@
 #!/usr/bin/env node
 /* eslint no-console: 0 */
 
-const cp = require('child_process');
+const execSync = require('child_process').execSync;
+const spawn = require('cross-spawn');
+const path = require('path');
 
-const TARGET_DIR = process.argv[2]; 
-const CLEANUP_DIRS = ['docs', '.circleci', '.github', '.git', '.npmignore', 'tasks'];
+let TARGET_DIR;
+const CLEANUP_DIRS = [
+  'docs',
+  '.circleci',
+  '.github',
+  '.git',
+  '.npmignore',
+  'tasks'
+];
 
 console.log('-------------------------------------------------------');
 console.log('Welcome to Create Evergreen App ♻️');
 console.log('-------------------------------------------------------');
 
-// overwrite existing directory or not?
-cp.execSync(`rm -rf ${TARGET_DIR}`);
+const checkTargetDir = appDir => {
+  if (appDir) {
+    TARGET_DIR = appDir;
+  } else {
+    console.error(
+      'Missing Project Directory! Please specifiy the application name e.g. create-evergreen-app my-app'
+    );
+    process.exit(1); // eslint-disable-line no-process-exit
+  }
+};
 
-// clone CEA from github
-cp.execSync(`git clone https://github.com/ProjectEvergreen/create-evergreen-app --branch release ${TARGET_DIR}`);
+function install() {
+  return new Promise((resolve, reject) => {
+    const command = 'npm';
+    const args = ['install', '--save', '--save-exact', '--loglevel', 'error'];
+    const child = spawn(command, args, { stdio: 'inherit' });
 
-// clean up the new repo directory
-CLEANUP_DIRS.forEach((directory) => {
-  cp.execSync(`rm -rf ${TARGET_DIR}/${directory}`);
-});
+    child.on('close', code => {
+      if (code !== 0) {
+        reject({
+          command: `${command} ${args.join(' ')}`
+        });
+        return;
+      }
+      resolve();
+    });
+  });
+}
 
-console.log('-------------------------------------------------------');
-console.log('Success, your project is ready to go!');
-console.log(`Just run: cd ${TARGET_DIR}`);
-console.log('And then: npm install && npm start (or use yarn)');
-console.log('-------------------------------------------------------');
+const run = async () => {
+  try {
+    // check params
+    checkTargetDir(process.argv[2]);
+
+    // overwrite existing directory or not?
+    execSync(`rm -rf ${TARGET_DIR}`);
+
+    // clone CEA from github
+    execSync(
+      `git clone https://github.com/ProjectEvergreen/create-evergreen-app --branch release ${TARGET_DIR}`
+    );
+    // clean up the new repo directory
+    CLEANUP_DIRS.forEach(directory => {
+      execSync(`rm -rf ${TARGET_DIR}/${directory}`);
+    });
+    // change directory to new app directory
+    process.chdir(path.resolve(process.cwd(), TARGET_DIR));
+    console.log('Installing dependencies...');
+    await install();
+    console.log('-------------------------------------------------------');
+    console.log('Success, your project is ready to go!');
+    console.log(`Just run: cd ${TARGET_DIR}`);
+    console.log('And then: npm install && npm start (or use yarn)');
```

Comment:
can any of these lines here or in the README be changed / updated now that we are doing the installation and `cd` for the user?